examples/binary-operators/binary_operators.bal

```diff
@@ -0,0 +1,67 @@
+import ballerina/io;
+
+public function main() {
+    int a = 10;
+    int b = 20;
+
+    // Addition
```

Comment:
These comments aren't good enough. We can mention at least what is expected as the operands.